Story/Episode 5/st_056100.csv

```diff
@@ -1,15 +1,15 @@
-st_056100_0010#0,"""Guided by its Divine Queen Margareta,<br>Harriet and the Guardian made their way towards the Holy Kingdom of Epic."""
-st_056100_0020#0,"""Both the fortresses and the villages on the border<br>were completely devoid of Epic's civilians<br>with only the traces of a hasty evacuation left."""
-st_056100_0030#0,"""As the people were evacuated, the damage to property and<br>harm to them were minimal. However, Margareta was still torn<br>that she had to force her people to evacuate their homes."""
-st_056100_0040#0,"""And yet she remained firm that if the problem were solved,<br>all would be able to be restored to how it was."""
+st_056100_0010#0,"""Guided by its Divine Queen, Margareta, Harriet and the Guardian<br>made their way towards the Holy Kingdom of Epic."""
+st_056100_0020#0,"""Both the fortresses and the villages on the border<br>were completely devoid of Epic's civilians,<br>with only the traces of a hasty evacuation left."""
+st_056100_0030#0,"""As the people had evacuated, the damage to property and<br>harm to them were minimal. However, Margareta was still torn<br>that she had to force her people to evacuate their homes."""
+st_056100_0040#0,"""And yet she remained firm that if the problem were<br>solved, all would be able to be restored to how it was."""
```

Comment:
I don't like this move here, makes more sense to have each line be it's own clause

Reply:
I'd agree if the lines were centered, but as they're left-aligned it looks kind of bad if they're all hugely different lengths.